src/utils/defaultSxProp.tsx

```diff
@@ -0,0 +1,3 @@
+import {BetterSystemStyleObject} from '../sx'
+
+export const defaultSxProp: BetterSystemStyleObject = {}
```

Comment:
we should use this more often instead of adding inline

`sx = {}` parameters, so I extracted and shared betwen each of the button components touched in this pr

Reply:
we could do that.  We could also, probably just mark the default as a readonly type (which is the effect freeze has here).  

Happy to check that out to see

Reply:
I'd imagine any change to this object, anywhere would cause a bunch of test cases to fail, or be pretty easy to pick up in code review.  we could make one per module, instead of sharing a single default, but seems common enough where a single object made sense